vnext/Desktop/React.Windows.Desktop.vcxproj

```diff
@@ -314,6 +317,9 @@
     <Import Project="$(SolutionDir)packages\Microsoft.Windows.CppWinRT.2.0.200615.7\build\native\Microsoft.Windows.CppWinRT.targets" Condition="Exists('$(SolutionDir)packages\Microsoft.Windows.CppWinRT.2.0.200615.7\build\native\Microsoft.Windows.CppWinRT.targets')" />
     <Import Project="$(V8_Package)\build\native\ReactNative.V8JSI.Windows.targets" Condition="Exists('$(V8_Package)\build\native\ReactNative.V8JSI.Windows.targets') AND '$(USE_V8)' == 'true'" />
     <Import Project="$(SolutionDir)packages\ReactNative.Hermes.Windows.0.1.6\build\native\ReactNative.Hermes.Windows.targets" Condition="Exists('$(SolutionDir)packages\ReactNative.Hermes.Windows.0.1.6\build\native\ReactNative.Hermes.Windows.targets')" />
+    <Import Project="..\packages\Microsoft.Build.Tasks.Git.1.0.0\build\Microsoft.Build.Tasks.Git.targets" Condition="Exists('..\packages\Microsoft.Build.Tasks.Git.1.0.0\build\Microsoft.Build.Tasks.Git.targets')" />
```

Comment:
>.. [](start = 21, length = 2)

Please use the $(SolutionDir)

Reply:
The same is for other project files.

---
In reply to: [456013866](https://github.com/microsoft/react-native-windows/pull/5533#discussion_r456013866) [](ancestors = 456013866)

Reply:
Sigh....  @microsoft/vcpp I hope this will be fixed in the next version of msvc/vs/msbuild..

Reply:
Note that I'll have to move the .props usages down then as well since SolutionDir is only created after `Microsfot.Cpp.Defaults.Props`

Reply:
Note we have been pretty inconsistent on SolutionDir. I'll send another PR to fix up the rest...